index.html

```diff
@@ -80,6 +83,72 @@ <h1 class="main-description">
           </li>
         </ul>
     </section>
+    <section class="project">
+    <div class="card">
+      <img class="pic" src="img/Snapshoot Portfolio.png" alt="First-img" />
+      <p class="card-name">Tonic</p>
```

Comment:
Good job for implementing the same font, but it's highly recommended to use header tags for the titles, so please make sure to put the project name into a `h[1-6]` tag.